stage0/run.go

```diff
@@ -258,9 +258,11 @@ func Run(cfg RunConfig, dir string) {
 
 	if cfg.SpawnMetadataService {
 		log.Print("Launching metadata svc")
-		if err := launchMetadataService(cfg.Debug); err != nil {
-			log.Printf("Failed to launch metadata svc: %v", err)
-		}
+		go func() {
+			if err := launchMetadataService(cfg.Debug); err != nil {
```

Comment:
@jonboulle you're right. It should be a critical error now. launchMetadataService used to use socket activation protocol so it could fail if metadata service was already running and that should not have been a critical error.